lib/request_handler/body_parser.rb

```diff
@@ -2,76 +2,35 @@
 
```

Comment:
I am not totally sure if we should get rid of the body_parser. We would then have 1 additional method + error handling in `base.rb`